acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/AcceptanceTestBase.java

```diff
@@ -90,7 +91,7 @@ protected TekuDepositSender createTekuDepositSender(final String networkName) {
   }
 
   protected BesuNode createBesuNode() {
-    return createBesuNode(config -> {});
+    return createBesuNode(Config::withMiningEnabled);
```

Comment:
Probably better to just change the default value in BesuConfig and provide a `withMiningEnabled(boolean)` so it can be disabled if needed. Otherwise switching to the other version of this method to configure some extra fields suddenly disables mining which will be hard to understand.